index.bs

```diff
@@ -3249,6 +3319,7 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     1. Run [=obtain and deliver debug reports on source registration=] with
         « "<code>[=source debug data type/source-storage-limit=]</code>" » and |source|.
     1. Return.
+1. [=Remove or update sources with incompatible attribution scope fields=] with |source|.
```

Comment:
Shouldn't this be checked after `delete sources for unexpired destination limit`?

Reply:
Done.